unison-src/transcripts/merge.output.md

```diff
@@ -46,11 +46,11 @@ bar = "bobs bar"
 Merge result:
 
 ``` ucm
-project/alice> merge /bob
```

Comment:
If possible to put the 500 line rename into at least a separate commit, it will make reviewing easier.